hasura/graphql-engine
# adds postgres password to docker-compose setup (fix #3894)

# Description

This just adds the `POSTGRES_PASSWORD` env var to the docker-compose file linked in the docs [here](https://docs.hasura.io/1.0/graphql/manual/getting-started/docker-simple.html).

### Affected components

### Solution and Design

- Add the env var. 

### Steps to test and verify

- To verify the bug, see the #3894 (just copy the docker-compose file and run it)

I have the following docker and docker-compose versions:

```
> docker --version
Docker version 17.12.0-ce, build c97c6d6
> docker-compose --version
docker-compose version 1.25.4, build unknown
```

-- edit:

feel free to reject if this isn't necessary; i imagine this file worked for someone once; maybe the postgres image should be tied to a version, instead?
